dothething/dothething/Assets.xcassets/Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "obama.jpg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
The changes in this file add a new image asset named "Placeholder" with three different scales (1x, 2x, and 3x). This is a common practice for managing assets in iOS applications to ensure proper display on devices with different screen resolutions.

The changes look correct and follow the standard structure for an image asset in Xcode. No further comments are needed for this file.